src/components/KExternalLink/KExternalLink.vue

```diff
@@ -10,6 +10,7 @@
     <ExternalLinkIcon
       v-if="!hideIcon"
       :size="KUI_ICON_SIZE_30"
+      title="External link icon"
```

Comment:
issue: if we add a title, that would mean upon hovering over the icon, the title attribute text is going to be visible in the UI.

I believe this should be `aria-description` or something instead since hovering over a link is common

Reply:
I guess since assistive technologies would announce the link anyway, maybe communicating that it is an external link through icon is over complicating. Let's give it `decorative` prop instead.